release_notes/41-release-notes.adoc

```diff
@@ -0,0 +1,439 @@
+
+- xref:../release_notes/41-release-notes.adoc#improvements-for-sensor-resync-general-availability_release-notes-41[Improvements for Sensor resync (General Availability)]
+
+
+[id="new-features_{context}"]
+== New features
+
+[id="manual-renewal-of-central-and-sensor-certificates_{context}"]
+=== Manual renewal of Central and Sensor certificates
+
+You now have the ability to renew certificates for {product-title-short} components when security breaches occur or when you want to renew certificates on your preferred schedule. With {product-title-short} 4.1, you can renew certificates for Central, Scanner, and `scanner-db` directly from the user interface (UI). This ensures continuous and secure communication between Central and all the Sensor components.
+
+Previously, you would receive an information banner notifying you when the Central certificate expired. Additionally, you were provided with new certificates for Central along with instructions for renewing Scanner and `scanner-db` certificates.Only users with administrative access to {product-title-short} can initiate the renewal and replacement of these certificates.
+
+
+[id="launching-vulnerability-management-20-technology-preview_{context}"]
+=== Launching Vulnerability Management 2.0 (Technology Preview)
+
+In {product-title-short} 4.1, you can experience significant improvements in Vulnerability Management with the introduction of Workload CVEs (Technology Preview). This feature is the first step towards Vulnerability Management 2.0 and provides you with improved workflows that are purposeful and effortless to navigate.
+
+These improved workflows provide actionable data with just a few clicks to help you address vulnerabilities. It is anticipated that Vulnerability Management 1.0 functionality will gradually transition to Vulnerability Management 2.0 in upcoming releases.
+
+This release provides a new navigation menu item, Vulnerability Management 2.0, which currently provides a view of vulnerabilities in applications running on clusters in your system. It is anticipated that in upcoming releases, this menu item will also provide a view of vulnerabilities present in platform and nodes.
+
+The *Workload CVEs* page provides a view of vulnerabilities in images and deployments, as well as options to filter and sort the data displayed. You can select specific vulnerabilities to view additional details, such as the Dockerfile layer in which the vulnerability exists and whether or not it can be fixed.
+
+In this technology preview, you can explore the read-only *Workload CVEs* page. However, upcoming releases are expected to include additional functionality such as Reporting, Risk Acceptance, and Manage Watched Images functionality. For more information, see xref:../operating/manage-vulnerabilities/vulnerability-management.adoc#vulnerability-management-process_acs-operating-manage-vulnerabilities[Vulnerability Management process].
```

Comment:
https://redhat-documentation.github.io/supplementary-style-guide/#statements-about-the-future
hey, have I misunderstood this section?

Reply:
The way I understand it is that we can refer to future plans when deprecating features or functions. (Like in the note in the section that you linked.) But we shouldn't be making specific claims about providing certain features in the future, because plans change and those features could not be implemented, or shelved. That isn't spelled out in the guidelines, just based on my experience here and in previous companies. So I just removed the specific "We anticipate providing X in a future release" and just referred to future enhancements in general.